webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/checks/CheckRudderGlobalParameter.scala

```diff
@@ -0,0 +1,124 @@
+* Copyright 2020 Normation SAS
+*************************************************************************************
+*
+* This file is part of Rudder.
+*
+* Rudder is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* In accordance with the terms of section 7 (7. Additional Terms.) of
+* the GNU General Public License version 3, the copyright holders add
+* the following Additional permissions:
+* Notwithstanding to the terms of section 5 (5. Conveying Modified Source
+* Versions) and 6 (6. Conveying Non-Source Forms.) of the GNU General
+* Public License version 3, when you create a Related Module, this
+* Related Module is not considered as a part of the work and may be
+* distributed under the license agreement of your choice.
+* A "Related Module" means a set of sources files including their
+* documentation that, without modification of the Source Code, enables
+* supplementary functions or services in addition to those offered by
+* the Software.
+*
+* Rudder is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
+
+*
+*************************************************************************************
+*/
+
+package bootstrap.liftweb.checks
+
+import bootstrap.liftweb.BootstrapChecks
+import com.normation.box._
+import zio._
+import zio.syntax._
+import better.files._
+import bootstrap.liftweb.BootstrapLogger
+import com.normation.errors.IOResult
+import com.normation.errors.Inconsistency
+import com.normation.eventlog.ModificationId
+import com.normation.rudder.domain.parameters._
+import com.normation.rudder.repository.RoParameterRepository
+import com.normation.rudder.repository.WoParameterRepository
+import com.normation.utils.StringUuidGenerator
+import net.liftweb.json._
+import com.normation.rudder.domain.eventlog._
+import com.normation.zio.ZioRuntime
+
+
+/**
+ *
+ * When the application is first initialized, we want to set the
+ * content of configuration-repository file to a consistant state,
+ * especially for directives/rules/groups, where we want to have
+ * all system categories and entities saved (else, we are going
+ * to have some surprise on the first import).
+ *
+ * So, if a full export wasn't done until know, just do one.
+ *
+ */
+class CheckRudderGlobalParameter(
+    roParamRepo: RoParameterRepository
+  , woParamRepo: WoParameterRepository
+  , uuidGen    : StringUuidGenerator
+) extends BootstrapChecks {
+
+  val resource = "rudder-system-global-parameter.conf"
```

Comment:
we could make it overridable, as a poor man solultion to replace it

Reply:
ha wait, it's simply to restore defaut, so no need to have such cleverness, you can ignore that